etc/nginx-conf-inner.in

```diff
@@ -53,3 +53,13 @@ add_header X-XSS-Protection "1; mode=block";
 
 error_log /var/log/nginx/domjudge.log;
 access_log /var/log/nginx/domjudge.log;
+
+# When running a proxy or loadbalancer in front of this nginx instance,
+# you can use these directives to trust the proxy / loadbalancer and let
+# this nginx instance use the correct client IP
+
+# The header to get the client IP from
+# real_ip_header X-Forwarded-For;
+# The IP address of the proxy / loadbalancer. Use this directive multiple
+# times if you need to use more than one IP address
+# set_real_ip_from 1.2.3.4;
```

Comment:
http://nginx.org/en/docs/http/ngx_http_realip_module.html

You can add one more sentence.

If there are multiple proxys, should be set `real_ip_recursive on;`

Reply:
I wonder if we really want that. You only need it if every proxy **adds** an IP to the header. If every proxy **replaces** the IP we should not be needing it, right? I'd say people who know proxies and/or use multiple should be smart enough to add it? I also wonder what a good comment would be for people to understand it.

Or do you disagree?

Reply:
What you said makes sense.